03_launch_mgmt_cluster.sh

```diff
@@ -546,10 +546,9 @@ clone_repos
 create_clouds_yaml
 if [ "${EPHEMERAL_CLUSTER}" != "tilt" ]; then
   start_management_cluster
+  kubectl create namespace metal3
```

Comment:
Does this line need to be inside the if statement for v1alpha3 ?

Reply:
its not right? its outside already